6-Final-Proj/server/tests/put_todos.test.ts

```diff
@@ -0,0 +1,56 @@
+import { Testkit } from "./testkit";
+import axios, { AxiosError, AxiosResponse } from "axios";
+import { TodoItem } from "../../common/types";
+import { aRandomItem, aUserId, aRandomPartialItem } from "./utils";
+
+const testkit = new Testkit();
+
+describe("On PUT /todos", () => {
+  testkit.beforeAndAfterEach();
+
+  it("should return 200 and the item should be updated \
+  when the item exists", async () => {
+    const userId = aUserId();
+    const todo1: TodoItem = aRandomItem({});
```

Comment:
do u need to pass something? meaning u can't do `aRandomItem()`?
if not then make it an optional param